src/routes/(admin)/questionbank/+page.server.ts

```
@@ -1,9 +1,15 @@
 import { BACKEND_URL } from '$env/static/private';
 import { getSessionTokenCookie } from '$lib/server/auth';
+import { getRequestEvent } from '$app/server';
```

ESLint

[error] 'getRequestEvent' is defined but never used.

(@typescript-eslint/no-unused-vars)